### ISSUE TITLE
Error when checking words via cli

### ISSUE THREAD

**Issue by Devmacs** [justinwilaby/spellchecker-wasm#26]
I'm on the tip of master. When I use the cli to check words, I get :

Ready
titel
/home/Devmacs/.nvm/versions/node/v13.3.0/lib/node_modules/spellchecker-wasm/lib/SuggestedItem.js:21
        return (this.cache.term = SuggestedItem.decodeString(this.data.slice(start, end)));
                                                ^

TypeError: SuggestedItem.decodeString is not a function
    at SuggestedItem.get term [as term] (/home/Devmacs/.nvm/versions/node/v13.3.0/lib/node_modules/spellchecker-wasm/lib/SuggestedItem.js:21:49)

Maybe the node module just needs a rebuild? I see the decodeString definition in both src/js/browser and src/js/nodejs.

**Comment by justinwilaby** [comment]
Is this still an issue after #28 ?

**Comment by Devmacs** [comment]
Yes. 

It should be using the nodejs decodeString, right?

**Comment by justinwilaby** [comment]
`SuggestedItem.decodeString` is assigned when `SpellcheckerWasm` is constructed.

It looks like the fix is to either nix the worker from the cli and use `SpellcheckerWasm` directly or make the assignment somewhere in `bin/spellchecker`.

```js
// bin/spellchecker
SuggestedItem.decodeString = (data: Uint8Array) => Buffer.from(data).toString('utf8');
```

**Comment by Devmacs** [comment]
Ok, I can take care of it. I wasn't sure if the code duplication would be acceptable.

**Comment by justinwilaby** [comment]
I'd prefer getting the worker out of the picture in the cli. It would simplify the code and increase startup time.

**Comment by Devmacs** [comment]
Should the SpellcheckerWorker.spec.ts test be removed?

Since the cli will use the nodejs Spellchecker, which is built from the same source as the browser Spellchecker (which is tested), I feel like the test may be redundant.

**Comment by justinwilaby** [comment]
Yes, this spec can be removed. Unit testing the cli should probably be done using `spawn` then sending commands via `stdin`.  If necessary, I can set this up on another PR.

**Comment by justinwilaby** [comment]
closed by #29
